wremnants/datasets/datagroups.py

```diff
@@ -504,16 +505,16 @@ def setGenAxes(self, gen_axes=None):
             gen_axes = [gen_axes]
 
         if gen_axes != None:
-            self.gen_axes = gen_axes
+            self.gen_axes = [x for x in gen_axes] # don't just copy gen_axes, can modify the object outside this function
```

Comment:
If it's already a list you can copy it gen_axes.copy(), if it's not a list you can convert it to a list list(gen_axes) which does a copy